ansible/roles/systemd/tasks/main.yml

```
@@ -0,0 +1,43 @@
+    dest: "/etc/systemd/system/{{ project_name }}.socket"
+    mode: '0644'
+  when: use_systemd_socket | default(false) | bool
+  notify:
+    - reload systemd
+    - restart application
+
+- name: Start and enable application service
+  systemd:
+    name: "{{ project_name }}"
+    state: started
+    enabled: yes
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)